packages/repl/package.json

```diff
@@ -5,38 +5,60 @@
 		"dev": "vite dev",
 		"build": "vite build",
 		"package": "svelte-package",
+		"watch": "svelte-package --watch",
 		"preview": "vite preview",
 		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
 		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. ."
 	},
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "^2.0.0",
-		"@sveltejs/kit": "^1.5.2",
-		"eslint": "^8.33.0",
-		"eslint-config-prettier": "^8.6.0",
+		"@sveltejs/kit": "^1.15.5",
+		"@sveltejs/package": "^2.0.2",
+		"eslint": "^8.38.0",
+		"eslint-config-prettier": "^8.8.0",
 		"eslint-plugin-svelte3": "^4.0.0",
-		"prettier": "^2.8.4",
-		"prettier-plugin-svelte": "^2.9.0",
-		"svelte": "^3.55.1",
-		"svelte2tsx": "^0.6.1",
-		"typescript": "^4.9.5",
-		"vite": "^4.1.1"
+		"prettier": "^2.8.7",
+		"prettier-plugin-svelte": "^2.10.0",
+		"svelte": "^3.58.0",
+		"svelte2tsx": "^0.6.11",
+		"typescript": "^5.0.4",
+		"vite": "^4.2.1"
 	},
 	"type": "module",
-	"svelte": "index.js",
+	"peerDependencies": {
+		"svelte": "^3.50.0"
+	},
 	"dependencies": {
-		"@rollup/browser": "^3.15.0",
-		"@sveltejs/site-kit": "3.2.2",
+		"@rollup/browser": "^3.20.2",
+		"@sveltejs/site-kit": "4.1.1",
 		"acorn": "^8.8.2",
-		"codemirror": "5.65.1",
+		"codemirror": "5.65.12",
+		"esm-env": "^1.0.0",
 		"estree-walker": "^3.0.3",
-		"resolve.exports": "^2.0.0",
+		"resolve.exports": "^2.0.2",
 		"svelte-json-tree": "^1.0.0",
 		"yootils": "^0.3.1"
 	},
 	"license": "MIT",
 	"publishConfig": {
-		"access": "public",
-		"directory": "package"
-	}
+		"access": "public"
+	},
+	"exports": {
+		"./package.json": "./package.json",
+		".": {
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js",
+			"default": "./dist/index.js"
+		}
+	},
+	"typesVersions": {
```

Comment:
might not need this?

Reply:
For people using with TS 4, still might need it